csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html

```diff
@@ -50,9 +54,7 @@ <h2><a href="https://www.eventbrite.ca/e/mountain-madness-2021-tickets-139106494
 				<div class='grass'></div>
 				<div class='grass'></div>
       </div>
-
     </section>
-
     <section id='info-container'>
```

Comment:
please add `<h2>F.A.Q.</h2>` right above this section tag. I am curious if that tags away the warning.